packages/react-components/src/checkbox/tests/chromatic/shared-stories.jsx

```diff
@@ -11,23 +11,23 @@ function Icons({ element, ...rest }) {
 
     return (
         <>
-            <div className="flex mb8">
+            <div className="flex items-start mb8">
```

Comment:
This is a detail, but can you use `ìtems-end` instead? I used `items-end` over `items-start` almost everywhere.